### PR TITLE
Uninstall unused Webpack 4 packages

### PR DESCRIPTION
**Pull request by anatoliik-lyft**
## Summary

 - Remove unused webpack 4 dependencies

## How did you test this change?

 - Ran `yarn test --prod`
 - Ran `yarn test`

## Related PRs:
 - https://github.com/facebook/react/pull/26887

### CODE REVIEW

File: packages/react-devtools-extensions/build.js [hunk 1]
```diff
@@ -56,7 +56,14 @@ const build = async (tempPath, manifestPath) => {
   const binPath = join(tempPath, 'bin');
   const zipPath = join(tempPath, 'zip');
 
-  const webpackPath = join(__dirname, 'node_modules', '.bin', 'webpack');
+  const webpackPath = join(
+    __dirname,
+    '..',
```
Comment (anatoliik-lyft) [inline packages/react-devtools-extensions/build.js:61]: changed it back as it was before https://github.com/facebook/react/pull/26887/files#diff-b6d5a082026c8873e9e518725027610b6936836db8bd4af2a7fa15fbdb563b7eL59

**Comment by anatoliik-lyft** [comment]
👋 @hoxyq tagging you in case you want to take a look, this PR is to replace webpack 4 in the root with webpack 5 and use the root one from workspaces. Would be awesome if you can take a look to make sure I didn't brake anything 🙌

**Comment by hoxyq** [comment]
> 👋 @hoxyq tagging you in case you want to take a look, this PR is to replace webpack 4 in the root with webpack 5 and use the root one from workspaces. Would be awesome if you can take a look to make sure I didn't brake anything 🙌

Hey, thanks for working on this!

I am not sure this is what we want though. These are devDependencies of specific packages inside the monorepo, `react-devtools-*` in this case. These are not dependencies of the `react` monorepo itself.

It is also questionable why do we need `webpack` listed here in root `package.json`, need to check if we use `webpack` somewhere from root `node_modules`. If not, then we can safely remove it.

**Comment by hoxyq** [comment]
> > 👋 @hoxyq tagging you in case you want to take a look, this PR is to replace webpack 4 in the root with webpack 5 and use the root one from workspaces. Would be awesome if you can take a look to make sure I didn't brake anything 🙌
> 
> Hey, thanks for working on this!
> 
> I am not sure this is what we want though. These are devDependencies of specific packages inside the monorepo, `react-devtools-*` in this case. These are not dependencies of the `react` monorepo itself.
> 
> It is also questionable why do we need `webpack` listed here in root `package.json`, need to check if we use `webpack` somewhere from root `node_modules`. If not, then we can safely remove it.

Also, having this dependency on a root level introduces a new constraint: imagine updating `webpack` to a new major version, say 6, you would need to check that this change doesn't break dozens of packages that might depend on `webpack`.

**Comment by hoxyq** [comment]
cc @sebmarkbage

Can we remove this root level `webpack` dependency or am I missing something? This was added in https://github.com/facebook/react/pull/17379

**Review by hoxyq — APPROVED** [review]
Looks safe to me.

**Pull request merged**